.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Unexpected behavior? Tell us more
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+# Describe your issue
+
+(Tell us what the problem is in a few word)
+
+# Your environment
+
+Please provide the following informations:
```

Comment:
```suggestion
Please provide the following information:
```

Reply:
Fixed it in https://github.com/inmanta/inmanta/pull/2416/commits/47d514a34f85f514a40a54849bef9082cb18ade9